app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Api
+  module V1
+    class PasswordsController < DeviseTokenAuth::PasswordsController
+      def edit
+        # if a user is not found, return nil
+        @resource = resource_class.with_reset_password_token(resource_params[:reset_password_token])
+
+        if @resource && @resource.reset_password_period_valid?
+          token = @resource.create_token unless require_client_password_reset_token?
+
+          # ensure that user is confirmed
+          @resource.skip_confirmation! if confirmable_enabled? && !@resource.confirmed_at
+          # allow user to change password once without current_password
```

Comment:
I think that these comments are a little bit redundant

Reply:
the comments are getting from the devise_token_auth directly, but this comment will be delete